latch_cli/docker_utils/__init__.py

```diff
@@ -0,0 +1,212 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+from textwrap import dedent
+from typing import List
+
+import yaml
+
+from latch_cli.constants import latch_constants
+from latch_cli.types import LatchWorkflowConfig
+
+
+class DockerCmdBlockOrder(Enum):
+    FIRST = auto()
+    PRECOPY = auto()
+    POSTCOPY = auto()
+    LAST = auto()
+
+
+@dataclass
+class DockerCmdBlock:
+    comment: str
+    commands: List[str]
+    order: DockerCmdBlockOrder
+
+
+def get_prologue(config: LatchWorkflowConfig) -> List[str]:
+    return [
+        "# latch base image + dependencies for latch SDK -- removing these will break the workflow",
```

Comment:
```suggestion
        "# latch base image + dependencies for latch SDK --- removing these will break the workflow",
```
look, em dash = `---`, minus/whatever-dash = `-`

`--` = en dash, which makes ABSOLUTELY no sense pretty much anywhere
so if you don't really care about the rules just use `-` everywhere. `--` is just offensive